xcell/mappers/mapper_base_Planck.py

```diff
@@ -0,0 +1,85 @@
+import healpy as hp
+import pymaster as nmt
+from .mapper_base import MapperBase
+
+
+class MapperBasePlanck(MapperBase):
+    def __init__(self, config):
+        self._get_Planck_defaults(config)
+        return
```

Comment:
these empty returns are not necessary

Reply:
done

Reply:
The return seems to still be there. Did you push?

Reply:
I forgot done now!